Sources/SwiftParser/Patterns.swift

```diff
@@ -214,20 +225,8 @@ extension Parser {
   /// for-in loops and guard clauses.
   mutating func parseMatchingPattern(context: PatternContext) -> RawPatternSyntax {
     // Parse productions that can only be patterns.
-    switch self.at(anyIn: MatchingPatternStart.self) {
-    case (.var, let handle)?,
-      (.let, let handle)?,
-      (.inout, let handle)?:
-      let bindingSpecifier = self.eat(handle)
-      let value = self.parseMatchingPattern(context: .bindingIntroducer)
-      return RawPatternSyntax(
-        RawValueBindingPatternSyntax(
-          bindingSpecifier: bindingSpecifier,
-          pattern: value,
-          arena: self.arena
-        )
-      )
-    case (.is, let handle)?:
+    OuterSwitch: switch self.at(anyIn: MatchingPatternStart.self) {
```

Comment:
I don’t actually think we need the label. If we `break` in the nested `switch`, then it just executes to the end of the outer `switch`, which is the same as `break OuterSwitch`.

---

If we do need it: We currently all caps our labels, so for consistency, this should be `OUTER_SWITCH`.